src/main/public/js/select-toggled-content.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function () {
+  $('.timeline-row .form-control-select').on('change', function () {
+
+    var $content = $(this).next()
+
+    if (this.value) {
+      $content
+        .removeClass('js-hidden')
+        .attr('aria-hidden', 'false')
```

Comment:
A `js-hidden` hides element for both screenreaders and browsers so I do not think `aria-hidden` applies here. If something is removed from DOM it won't be visible to screen readers.